gpeople/Package.md

```diff
@@ -1,320 +1,21 @@
-# Ballerina Google People Connector 
-Connects to Google People using Ballerina.
+Connects to Google People API from Ballerina
```

Comment:
Should "Ballerina" be linked to the Ballerina site?

Reply:
As it is found across other connectors in similar way i added that link